www/admin/artworks/post.php

```diff
@@ -11,7 +11,7 @@
 try{
 	$artwork = Artwork::Get($artworkId);
 }
-catch(Exceptions\SeException $ex){
+catch(Exceptions\SeException){
```

Comment:
```suggestion
catch(Exceptions\SeException $_){
```

Reply:
The server is now on PHP 8.1.2 so you can catch exceptions without assigning them to a variable.

Reply:
Good to know - I'll see if I can get the docker image upgraded too.